uber/site_sections/registration.py

```diff
@@ -101,11 +101,15 @@ def form(self, session, message='', return_to='', omit_badge='', check_in='', **
                     attendee.checked_in = localized_now()
                 session.add(attendee)
                 if return_to:
-                    raise HTTPRedirect(return_to + '&message={}', 'Attendee data uploaded')
+                    raise HTTPRedirect(return_to + '&message={}', 'Attendee data saved')
                 else:
-                    raise HTTPRedirect('index?uploaded_id={}&message={}&search_text={}', attendee.id,
-                        '{} has been uploaded'.format(attendee.full_name),
-                        '{} {}'.format(attendee.first_name, attendee.last_name) if c.AT_THE_CON else '')
+                    msg_text = '{} has been saved'.format(attendee.full_name)
+                    return_to_search = 'save_return_to_search' in params['save'] if 'save' in params else False
```

Comment:
A better way to write this would be to just say

```python
if params.get('save') == 'save_return_to_search':
```

The reason being that ``params.get('save')`` will return ``None`` if it's not present, so this is always a safe comparison to make.